contrib/masmx86/bld_ml32.bat

```diff
@@ -1,2 +1,2 @@
-ml /coff /Zi /c /Flmatch686.lst match686.asm
-ml /coff /Zi /c /Flinffas32.lst inffas32.asm
+ml /coff /Zi /c /safeseh /Flmatch686.lst match686.asm
+ml /coff /Zi /c /safeseh /Flinffas32.lst inffas32.asm

```

Comment:
If it is recommended for older versions, I would think it should be separated as different patch set, so it can be validated separately by people who still use older Visual Studio version. The other choice would be dirty hack in my opinion as it would combine the two batch files and add detection for Visual Studio version and select code paths depending on it.